scripts/create-exawind-snapshot.sh

```diff
@@ -51,7 +51,7 @@ elif [[ "${SPACK_MANAGER_MACHINE}" == "summit" ]]; then
   cmd "nice -n19 spack manager snapshot -m -s exawind%gcc+hypre+cuda+amr_wind_gpu+nalu_wind_gpu exawind%gcc+hypre~cuda"
 elif [[ "${SPACK_MANAGER_MACHINE}" == "perlmutter" ]]; then
   NUM_CORES=8
-  cmd "nice -n19 spack manager snapshot -m -s exawind%gcc+hypre+cuda+amr_wind_gpu+nalu_wind_gpu"
+  cmd "nice -n19 spack -d manager snapshot -m -s exawind%gcc+hypre+cuda+amr_wind_gpu+nalu_wind_gpu"
```

Comment:
probably don't want to leave the `-d` in there.
```suggestion
  cmd "nice -n19 spack manager snapshot -m -s exawind%gcc+hypre+cuda+amr_wind_gpu+nalu_wind_gpu"
```

Reply:
Sounds good.